ui/tutorial-renderer/src/main/res/layout/tutorial_tips_start.xml

```diff
@@ -23,42 +23,7 @@
             app:layout_constraintEnd_toEndOf="@id/marginEnd"
             app:layout_constraintTop_toTopOf="parent" />
```

Comment:
This compose view doesn't currently have a bottom constraint specified, it probably needs to be specified.